kbs/docs/confidential-containers-eventlog.md

```diff
@@ -0,0 +1,142 @@
+│       │                     │       │      │                     │
+│       └───┬─────────┬───────┘       │      └─────────────────────┘
+│           │         │               │                             
+│           │         ▼               │                             
+│    Extend │     ┌──────┐            │                             
+│           │     │ AAEL │            │                             
+│           │     └──────┘            │                             
+│     ┌─────▼───────────────────┐     │                             
+│     │ Runtime Measurements/PCR│     │                             
+│     └─────────────────────────┘     │                             
+│                                     │                             
+│           Confidential VM           │                             
+└─────────────────────────────────────┘                               
+```
+
+In this specification, we provide detailed information on the Attestation Agent Event Log (AAEL) format in confidential computing scenarios.
+
+Currently, as the kernel does not offer a unified interface for maintaining Eventlogs [1], we have decided to temporarily use the AAEL. 
+
+Once the kernel releases a unified Eventlog format, we will update the next version of the specification to ensure that the existing AAEL Event Entries are compatible with the new Kernel Eventlog format. We will also strive to minimize any impact on the user experience for existing users.
+
+## Specifications
+
+AAEL is a standard for event logging over guest kernel level.
+
+### Attestation Agent Event Log (AAEL)
+
+1. Binding of AAEL with Dynamic Measurement Registers
+
+Entries recorded by AAEL are bound to a specific PCR register value. With each new event added, an extend operation is
+performed on the designated PCR register. When the platform provides a (v)TPM interface, the PCR register corresponds
+to the (v)TPM's PCR register. On a platform that is solely TEE, PCR is mapped to a specific [Confidential Computing event log Measurement Register (CCMR)](https://uefi.org/specs/UEFI/2.10/38_Confidential_Computing.html#virtual-platform-cc-event-log)
+according to platform-specific rules. This mapping ensures AAEL has applicability even outside TEE scenarios.
+
+2. AAEL Log Entry Format
+
+AAEL log entries consist of two types: Event Entry and INIT Entry. 
+
+__INIT Entry__ is recorded only once at the beginning of AAEL when AA first initializes, capturing the current value of a specific PCR, formatted as
+
+```
+INIT/<hash-algorithm> <hex-digest>
+```
+Where,
+- `<hash-algorithm>`: may be `sha256`, `sha384`, or `sha512`.
+- `<hex-digest>` is the base16-encoded PCR register value. The length MUST be aligned with the `<hash-algorithm>`. Padding with zeros or truncation MUST be applied if necessary to align with the digest length.
+
+__Event Entry__ records specific events in the format
+```
+<Domain> <Operation> <Content>
```

Comment:
Is this terminated by a newline character?

Reply:
Yes. With LF ('\n', '0x0A') without any CR ('\r', `0x0D`)

Reply:
Added detail